src/logic/calculate.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-param-reassign, eqeqeq, no-undef */
+import operate from './operate';
+
+const calculate = (buttonName, dataObj) => {
```

Comment:
According to the instructions on pathwright, you should make use of destructuring.
Example;  `{ total, next, operation } = dataObj `
![image](https://user-images.githubusercontent.com/31897434/120503820-86097700-c3bb-11eb-9f60-ce74199588ad.png)